src/Umbraco.Web/PropertyEditors/ValueConverters/MediaCropperValueConverter.cs

```diff
@@ -0,0 +1,267 @@
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using Umbraco.Core;
+using Umbraco.Core.Configuration;
+using Umbraco.Core.Logging;
+using Umbraco.Core.Models;
+using Umbraco.Core.Models.PublishedContent;
+using Umbraco.Core.PropertyEditors;
+using Umbraco.Core.PropertyEditors.ValueConverters;
+using Umbraco.Core.Services;
+using Umbraco.Web.Extensions;
+using Umbraco.Web.Models;
+
+namespace Umbraco.Web.PropertyEditors.ValueConverters
+{
+    /// <summary>
+    /// Used to strongly type the value for the media cropper
+    /// </summary>
+    [DefaultPropertyValueConverter(typeof(JsonValueConverter))] //this shadows the JsonValueConverter
+    public class MediaCropperValueConverter : PropertyValueConverterBase, IPropertyValueConverterMeta
+    {
+        private readonly IDataTypeService _dataTypeService;
+        private readonly IEntityService _entityService;
+
+        public MediaCropperValueConverter()
```

Comment:
Better use constructor overloading here:

```
public MediaCropperValueConverter()
    : this(ApplicationContext.Current.Services.DataTypeService, ApplicationContext.Current.Services.EntityService)
{ }
```

Reply:
So like this?

```

        private readonly IDataTypeService _dataTypeService;
        private readonly IEntityService _entityService;

        public MediaCropperValueConverter() : this(ApplicationContext.Current.Services.DataTypeService, ApplicationContext.Current.Services.EntityService)
        {
        }

        public MediaCropperValueConverter(IDataTypeService dataTypeService, IEntityService entityService)
        {
            if (dataTypeService == null) throw new ArgumentNullException("dataTypeService");
            if (entityService == null) throw new ArgumentNullException("entityService");
            _dataTypeService = dataTypeService;
            _entityService = entityService;
        }
```